DataFormats/HcalDetId/src/HcalZDCDetId.cc

```diff
@@ -1,6 +1,13 @@
 #include "DataFormats/HcalDetId/interface/HcalZDCDetId.h"
 #include "FWCore/Utilities/interface/Exception.h"
 
+static const int kZDCChannelMask   = 0xF;
```

Comment:
assigned values are not needed.
It's dangerous to repeat them in two separate files due to possible divergence in values in future developments.